### ISSUE TITLE
When creating WebAppContext without session-config and with NO_SESSIONS throws NPE

### ISSUE THREAD

**Issue by btd** [eclipse/jetty.project#1558]
If i create WebAppContext this way:
```scala
  val webContext = new WebAppContext(
      null, //the parent handler
      "/", //the context path
      null, // SessionHandler for this web app
      null, // SecurityHandler for this web app
      null, // ServletHandler for this web app
      null, //ErrorHandler for this web app
      ServletContextHandler.NO_SESSIONS|ServletContextHandler.NO_SECURITY
    )
    webContext.setWar(config.webDir)
```
It throws NPE, because MetaData tried to merge default web.xml with session-config (which set 30 min), even if my own web.xml do not contain session-config section.
So it looks there is no way to create WebAppContext without sessions.

**Comment by janbartel** [comment]
Fixed in jetty-9.4

**Comment by btd** [comment]
@janbartel Thank you
